core/ui/common/src/main/java/io/filmtime/core/ui/common/componnents/RatingsInfo.kt

```diff
@@ -36,87 +31,47 @@ fun RatingsInfo(
       verticalAlignment = Alignment.CenterVertically,
       horizontalArrangement = Arrangement.spacedBy(8.dp, Alignment.CenterHorizontally),
     ) {
-      ratings.imdb?.let {
-        Image(
-          modifier = Modifier.size(28.dp),
-          painter = painterResource(R.drawable.ic_imdb),
-          contentDescription = null,
+      ratings.imdb?.let { rate ->
+        RatingCard(
+          icon = R.drawable.ic_imdb,
+          rate = "${rate.rating}",
+          votes = ratings.imdb?.votes ?: "N/A",
+          onClick = rate.link?.let { { onRatingClick(rate.link!!) } },
```

Comment:
1. Why do not we handle the link click and hoist this url to parent composable while we can sort it out here?
2. Please try to avoid using !! as much as possible. Although in this case rate.link!! will never be null, it takes time for someone reading the code to understand why. They might need to verify whether a null case is possible and if an exception could be thrown. Using a safer approach can make the code more readable and maintainable.

```suggestion
          onClick = rate.link?.let { { onRatingClick(it) } },
```